### ISSUE TITLE
[DXVK 1.10.1] Witcher 3 crashes on startup - Possible regression?

### ISSUE THREAD

**Issue by jrugia** [doitsujin/dxvk#2570]
Attempting to run Witcher 3 with DXVK 1.10.1 crashes on startup. Reverting to DXVK 1.10 as the sole change fixes the issue and game works fine.

I've tried older Mesa versions (21.3.7), setting 'RADV_DEBUG=llvm', clearing the dxvk-cache file. In addition the fault occurs with other GPU's (RX 560 and RX 5500 XT).

### Software information
The Witcher 3

### System information
- GPU: RX 6800
- Driver: Mesa 22.0.0
- Wine version: 6.19
- DXVK version: 1.10.1

### Log files
- d3d11.log:  [witcher3_d3d11.log](https://github.com/doitsujin/dxvk/files/8374133/witcher3_d3d11.log)
- dxgi.log: [witcher3_dxgi.log](https://github.com/doitsujin/dxvk/files/8374139/witcher3_dxgi.log)

**Comment by Blisto91** [comment]
At first i thought i could reproduce, but that was another issue.
Anyway it seems to work fine for me with my R9 380 and mesa 22.0

**Comment by doitsujin** [comment]
Works fine on my end (6900XT, Mesa 22.0).

**Comment by iWeaker** [comment]
works fine  

![Captura de pantalla de 2022-03-29 20-00-28](https://user-images.githubusercontent.com/49424816/160742828-db04235d-f450-4d88-bcfb-2375a7477e56.png)

**Comment by duck-rh** [comment]
I'm experiencing the same problem with ELEX 2 on RADV (Radeon RX 5700 XT) with Mesa 22.0.0, Vulkan 1.3.204 and Wine 7.5:
```
0108:err:seh:NtRaiseException Unhandled exception code c0000005 flags 0 addr 0x1700329db
```
Reverting to 1.10.0 solves the problem. With 1.10.1 I don't even get any log file created even in debug, Setting `VK_INSTANCE_LAYERS` did not help either.

I also experience this with Saints Row - The Third, also using D3D11, same error, same fix.

I can start ELEX 1, also using D3D11, with 1.10.1.

**Comment by duck-rh** [comment]
I just rebuilt 1.10.0 on my system and I now can experience the same problem. My previous build was with libwine-development-dev 6.5 and now it used 6.14 (the Wine version I run the games with differ because the Wine Debian packaging lags behind way too much but dxvk is made to be uploaded in Debian). Other possibly meaningful difference with the working build:
  - meson 0.61.2 -> 0.62
  - I now have to use the fix in #2572
  - libz-mingw-w64 got a fix for CVE-2018-25032 but stayed at the same version
  - libvulkan1 1.3.204.0 -> 1.3.204.1
  - I ran the game with mesa 22 RC2

I made a rebuild with 6.5, changing nothing else, and now ELEX 2 works again, albeit very slowly (compared to my previous build).

**Comment by WinterSnowfall** [comment]
Not sure if anyone cares about the Nvidia side of things, but with Nvidia 510.60.02 + Wine Staging 7.5 + DXVK 1.10.1 it's all good here, I have no issues.

**Comment by duck-rh** [comment]
@doitsujin On my side I'm building with the current state of Debian unstable to upload new dxvk packages, nothing "custom". The only special thing in my setup is using winehq wine packages to run the games (but not to build dxvk, which is built in a clean build environment).

**Comment by jrugia** [comment]
It seems the fault was coming from my Wine build. I've compiled a fresh Vanilla Wine 7.0 build and game ran instantly.

Glad that was the issue, although I'm confused how come it was the only game affected as I have a few more games using the same Wine prefix.


![1](https://user-images.githubusercontent.com/13750621/160902934-725ef64e-01de-48c1-9c02-4ab394a89aab.jpg)


Closing ticket since this is a non DXVK issue.

**Comment by stefson** [comment]
I also had a crash at startup with >=dxvk-1.10.1 and wine-6.20, solved by upgrading to latest wine-7.5

when I diffed the tarballs, it seems to me that dxvk-1.10.1 now uses `cpp_std=c++17`, maybe that has uncovered a bug in the wine implementation.
